pkg/store/kotsstore/kots_store.go

```diff
@@ -194,10 +201,36 @@ func (s KOTSStore) IsNotFound(err error) bool {
 		return true
 	}
 
+	if kuberneteserrors.IsNotFound(err) {
```

Comment:
whooh, i thought we had this already, hope this didn't cause subtle bugs

Reply:
indeed.  but this feels like good time to add this.